src/software/backend/simulation/physics/physics_robot_test.cpp

```diff
@@ -522,8 +522,8 @@ TEST_F(PhysicsRobotTest, test_robot_drive_forward_at_angle)
     }
 
     auto robot = physics_robot.getRobotWithTimestamp(Timestamp::fromSeconds(0));
-    EXPECT_GT(robot.velocity().x(), 0.5);
-    EXPECT_GT(robot.velocity().y(), 0.5);
+    EXPECT_GT(robot.velocity().x(), 0.4);
```

Comment:
Will these values also have to be changed when the damping is changed back?

Reply:
Yes, ideally these damping values are passed by parameters so we can have fixed values for the tests, but I think that would get out of scope of this PR pretty quickly.

I'm not happy that I had to change values, but they were small enough I'm confident the test is still testing the same behavior (ie. I'm confident these numbers would correctly validate other values of damping or other small tweaks). I've made a note to look again when we do proper parameter injection.